packages/app/obojobo-repository/shared/components/repository-nav.test.js

```diff
@@ -169,4 +203,128 @@ describe('RepositoryNav', () => {
 		})
 		expectMenuToBeClosed(component)
 	})
+
+	test('loads notifications from cookies on mount', () => {
+		document.cookie =
+			'notifications=' +
+			JSON.stringify([{ key: 1, text: 'Test Notification', title: 'Test Title' }])
+
+		const component = create(<RepositoryNav {...navProps} />)
+		const tree = component.toJSON()
+
+		expect(tree).toMatchSnapshot()
+		expect(document.cookie).toBe(
+			'notifications=[{"key":1,"text":"Test Notification","title":"Test Title"}]'
+		)
+	})
+
+	test('renders null when document.cookie is null', () => {
+		const originalDocument = document.cookie
+		document.cookie = null
+
+		const reusableComponent = <RepositoryNav {...navProps} />
+		let component
+		act(() => {
+			component = create(reusableComponent)
+		})
+		const tree = component.toJSON()
+		expect(tree).toMatchSnapshot()
+
+		if (document && document.cookie) {
+			//don't get here
```

Comment:
We could probably have some `expect`s above this to check that `document` has a value but `document.cookie` does not, rather than this if/else.